ramls/schemas/GOBIMapping.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "GOBI DataMapping Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "fieldName" : {
+      "type" : "string",
+      "description": "name of the field to map"
+    },
+    "defaultValue" : {
+      "type" : "string",
+      "description": "default value of field"
+    },
+    "fieldPath" : {
+      "type" : "string",
+      "description" : "XPath to extract data from original record"
+    },
+    "fieldType" : {
```

Comment:
think we might need to allow for additional context/information to be passed in (method name, url, etc).  I could definitely see this happening in the Lookup case.  An alternative would be to remove "Lookup" from the enum and instead list the various methods available.  e.g. 'lookupVendorId'